integration/utils/tx_stats_tracker_test.go

```diff
@@ -40,7 +40,7 @@ func TestTxStatsTracker(t *testing.T) {
 	assert.InDelta(t, 10., st.TTS.Max(), 1.)
 }
 
-// TestTxStatsTrackerString tests the String() method.
+// TestTxStatsTrackerString tests the Name() method.
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig